smithy-aws-protocol-tests/build.gradle

```diff
@@ -38,3 +38,5 @@ smithy {
     format.set(false)
     smithyBuildConfigs.set(project.files())
 }
+
+sourceSets.main.java.srcDirs = ['model', 'src/main/smithy']
```

Comment:
IntelliJ's Smithy plugin prompted me to add this line, otherwise all the `.smithy` code was unparseable for it.